Resources/Computing/ComputingElement.py

```diff
@@ -40,10 +40,12 @@
 from DIRAC.Core.Utilities.Time                        import dateTime, second
 from DIRAC                                            import S_OK, S_ERROR, gLogger, version
 from DIRAC.Core.Utilities.ObjectLoader                import ObjectLoader
+from DIRAC.Core.Utilities.Os                          import getNumberOfCores
```

Comment:
??? What do you mean ? This is just a method to get the number of (logical) cores in the current node. There is a case that a node can be used entirely, e.g. VMs, but the actual number of processors is not specified precisely in the configuration. So, should be discovered in place